libs/algebra/test/multiexp.cpp

```diff
@@ -158,11 +158,12 @@ BOOST_AUTO_TEST_SUITE(multiexp_test_suite)
 
 BOOST_AUTO_TEST_CASE(multiexp_test_case) {
 
+    // TODO: These tests look like performance benchmarks. Should be removed from CI
     std::cout << "Testing BLS12-381 G1" << std::endl;
-    print_performance_csv<curves::bls12<381>::g1_type<>, curves::bls12<381>::scalar_field_type>(2, 12, 14, true);
+    print_performance_csv<curves::bls12<381>::g1_type<>, curves::bls12<381>::scalar_field_type>(2, 4, 6, true);
```

Comment:
Maybe we move this to bench_test folder?

Reply:
moved to benches